src/main/java/com/questhelper/skills/agility/AlKharid.java

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
I like the idea of having the QH arrow on the agility shortcut elements, but I'm not sure about having each step in the sidebar. What would you think of having in the sidebar just a single big step of 'Do the agility course in Al Kharid, starting on the north wall of the tannery' or w/e? This can be done with a parent ConditionalStep with the text, which is what's passed back to be displayed.

The individual step text could still appear on the overlay maybe, with a bit of a cleanup of the text to be more descriptive rather than just the use text of the obstacle course. Like 'Climb across the tightrope to the east.' and so on.